src/servers/Server_Zone/Actor/PlayerSql.cpp

```diff
@@ -316,6 +321,12 @@ void Core::Entity::Player::createUpdateSql()
    if( m_updateFlags & PlayerSyncFlags::HowTo )
       charaDetailSet.insert( " HowTo = UNHEX('" + std::string( Util::binaryToHexString( static_cast< uint8_t* >( m_howTo ), sizeof( m_howTo ) ) ) + "')" );
 
+   if ( m_updateFlags & PlayerSyncFlags::Title )
+   {
+      charaDetailSet.insert( " ActiveTitle = " + std::to_string( m_title ) );
+      charaDetailSet.insert( " TitleList = UNHEX('" + std::string( Util::binaryToHexString( reinterpret_cast< uint8_t* >( m_titleList ), sizeof( m_titleList ) ) ) + "')" );
```

Comment:
not sure the additional std::string is needed here if Utils::binaryToHexString returns one